pkg/route/api/v1/types.go

```diff
@@ -43,6 +43,10 @@ type RouteSpec struct {
 	// be defaulted to Service.
 	To kapi.ObjectReference `json:"to"`
 
+	// SecondaryServices is an extension of the 'to' field. If more than one service needs to be
+	// pointed to, then use this field
+	SecondaryServices []kapi.ObjectReference `json:"secondaryServices,omitempty"`
```

Comment:
The point of this field is to hold weights, so weights should be part of the API for each one.  The annotations are tangential to the point of the PR.